datasets/ethos/README.md

```diff
@@ -0,0 +1,180 @@
+---
+annotations_creators: [crowdsourced, expert-generated]
+language_creators: [found, other]
+languages: [en]
+licenses: [agpl-3.0-or-later]
+multilinguality: [monolingual]
+size_categories: [n<1K]
+source_datasets: [original]
+task_categories: [text-classification]
+task_ids:
+- binary: [sentiment-classification, text-classification-other-Hate Speech Detection]
+- multilabel: [multi-label-classification, sentiment-classification, text-classification-other-Hate
+      Speech Detection]
```

Comment:
@lhoestq  Dear Quentin! Thanks for merging this PR. I checked the https://huggingface.co/datasets/ethos page and show that the yaml tags are wrong. Then, I tried to find out why. I think this is happening due to this change you made. Because if you see the rendered ReadME.md with this code (the one I had originally) it renders the yaml table. I guess the error occurs in the task_ids in the version you made. 

What should I do? :/ 

'Original Code':
annotations_creators: [crowdsourced, expert-generated]
language_creators: [found, other]
languages: [en]
licenses: [agpl-3.0-or-later]
multilinguality: [monolingual]
size_categories: [n<1K]
source_datasets: [original]
task_categories: [text-classification]
task_ids:
- binary: [sentiment-classification, text-classification-other-Hate Speech Detection]
- multilabel: [multi-label-classification, sentiment-classification, text-classification-other-Hate
      Speech Detection]

Reply:
Well! Problem solved :D with this commit! https://github.com/huggingface/datasets/commit/856d46c41972e20104dfb9ec4d75b94513cc7a09! Thanks @lhoestq and @julien-c